oslabs-beta/Kaskade
# JW/ read and write data file with ipc, navbar styling, new session

# Summary

1. implemented proper write and read of data file using electron ipc
2. navbar styling with mui
3. add a new session button in session side bar

## Type of Change



## Testing Steps

1. Spin up electron app via command npm run dev.
2. Go to the sessions page, you should see existing sessions and a new session button
3. click new session, a new session should be created 
4. reload electron, the newly created session should persist


## Checklist
